handlers/main.yml

```diff
@@ -2,6 +2,10 @@
 - name: run riak_disk_tune
   command: /bin/bash /usr/local/bin/riak_disk_tune.sh
 
+- name: restart riak and ensure enabled
+  service: name=riak enabled=yes state=restarted
+  when: riak_init_system == 'system'
```

Comment:
Better not to change this to variable in notify, because of https://github.com/ansible/ansible/issues/19898